Mathlib/AlgebraicGeometry/Morphisms/QuasiSeparated.lean

```diff
@@ -183,63 +183,84 @@ theorem QuasiSeparated.of_comp {X Y Z : Scheme} (f : X ⟶ Y) (g : Y ⟶ Z) [Qua
       (pullbackRightPullbackFstIso g (Z.affineCover.map i) f).hom
   · exact inferInstance
 
+/--
+Specialize `restrictOpen` to `CommRingCat` because inferring `C := CommRingCat` isn't reliable.
+
+Instead of unfolding the definition, rewrite with `restrictOpenCommRingCat_apply` to ensure the
+correct coercion to functions is taken.
+-/
+abbrev _root_.TopCat.Presheaf.restrictOpenCommRingCat {X : TopCat}
```

Comment:
This should probably be upstreamed somewhere: `set_option linter.minImports` suggests it should go to the join of ` Mathlib.Topology.Sheaves.Presheaf` and `Mathlib.Algebra.Category.Ring.Basic`, which I am not familiar enough with this library to find easily.

(Also there are a couple instances in other files where I had to do `restrictOpen (C := CommRingCat)`, I ended up adding it here because this file needed the hint *a lot*.)

(Good `ImportGraph` feature idea?)